### ISSUE TITLE
return wiki to 95% service

### ISSUE THREAD

**Issue by WardCunningham** [WardCunningham/remodeling#1]
We'll develop a checklist here for restoring wiki serving static json to a client-side rendering engine. We'll be adding more as we launch this project.

- [x] paragraph with emphasis, [eg](http://c2.com/wiki/remodel/compare.html?TextFormattingRules)
- [x] internal and external links
- [x] bullet lists and their variations
- [x] plugin to existing routes with 302 redirects
- [x] rehosting from any static site without reconfiguration

**Comment by almereyda** [comment]
For the `plugin to existing routes with 301 redirects`, I suppose the missing search is already known?

Tried clicking [http:wiki?search=CategoryWiki](http://c2.com/wiki/remodel/wiki?search=CategoryWiki) on http://c2.com/wiki/remodel/?CategoryWiki without result.

**Comment by WardCunningham** [comment]
@almereyda I've postponed search until #2. Title search will be easy because I'm loading a full list of titles. Word search is more data intensive probably requiring a db server side.

**Comment by WardCunningham** [comment]
We reopened wiki sans some important services like search and about a 1000 old pages with character encoding problems. We've relocated to wiki.c2.com but will not permently redirect there until we have these and seo issues worked out.

I offer as celebration this old but still useful pattern along with a decade of refinements by some of wiki's most influential contributors. Click either link.

http://c2.com/cgi/wiki?SystemOfNames redirects to ⇒
http://wiki.c2.com/?SystemOfNames

As I am now overcome with nostalgia I will also acknowledge those who attended wiki's 21st birthday party six months ago. Read the [full story](http://ward.bay.wiki.org/wiki-turns-21.html).

![image](https://cloud.githubusercontent.com/assets/12127/19623417/865450d6-987d-11e6-9092-304fb508c82a.png)
